### PR TITLE
Two sync

### PR DESCRIPTION
**Pull request by inderpreetsingh**
This allows a two way sync between desktop and web.

_video and more details coming soon_

### CODE REVIEW

File: www/js/search.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const allowedKeys = [
 const sessionList = [];
 const sessionStatesList = {};
 const currentShabad = [];
-let currentShabadId = 0;
+let currentShabadState = 0;
```
Comment (tsingh777) [inline www/js/search.js:25]: maybe initialize this `{id: null, type: null}`? 

that way you won't get any "can't not access prop of" issues

File: www/js/toolbar.js [hunk 2]
```diff
@@ -94,6 +94,34 @@ const setListeners = () => {
             type: 'response-control',
             success: isPinCorrect,
           });
+
+          // if Pin is correct and there is a shabad already in desktop, emit that shabad details.
+          if (isPinCorrect) {
+            const currentShabad = global.core.search.getCurrentShabadId();
+            const currentVerse = document.querySelector(`#shabad .panktee.current`);
+            const homeVerse = document.querySelector(`#shabad .panktee.main`);
```
Comment (tsingh777) [inline www/js/toolbar.js:102]: I'm guessing updating the home line from desktop is out of scope right now?

Comment (maneetpaul) [reply www/js/toolbar.js:102]: Not too worried about that if we can't add it right now
